### PR TITLE
[pallet-revive] Add pallet to AH westend

### PR DESCRIPTION
**Pull request by pgherveou**
Add pallet-revive to Westend runtime, and configure the runtime to accept Ethereum signed transaction

### CODE REVIEW

**Comment by pgherveou** [comment]
bot help

**Comment by pgherveou** [comment]
bot bench cumulus-assets --Runtime asset-hub-westend --Pallet pallet_revive

**Comment by pgherveou** [comment]
bot bench cumulus-assets --Runtime asset-hub-westend --pallet pallet_revive

**Comment by pgherveou** [comment]
bot bench cumulus-assets --runtime asset-hub-westend --Pallet pallet_revive

**Comment by pgherveou** [comment]
bot bench cumulus-assets --runtime asset-hub-westend --pallet pallet_revive

**Comment by pgherveou** [comment]
bot bench cumulus-assets --runtime asset-hub-westend --pallet pallet_revive

**Comment by pgherveou** [comment]
bot cancel 19-653fcfbe-34a4-4fd4-8466-cc9da2f973f6

**Comment by pgherveou** [comment]
bot bench cumulus-assets --runtime asset-hub-westend --pallet pallet_revive

**Comment by pgherveou** [comment]
bot bench cumulus-assets --runtime asset-hub-westend --pallet pallet_revive

File: cumulus/parachains/runtimes/assets/asset-hub-westend/Cargo.toml
Comment (athei) [inline cumulus/parachains/runtimes/assets/asset-hub-westend/Cargo.toml:1]: In order to be able to run the benchmarks you need to forward the `riscv` feature. Hopefully we can soon switch to an upstream toolchain and get rid of this feature. I think we should just use the default weights until then.

File: cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs [hunk 1]
```diff
@@ -934,6 +935,52 @@ impl pallet_xcm_bridge_hub_router::Config<ToRococoXcmRouterInstance> for Runtime
...
+impl pallet_revive::Config for Runtime {
+	type Time = Timestamp;
+	type Currency = Balances;
+	type RuntimeEvent = RuntimeEvent;
+	type RuntimeCall = RuntimeCall;
+	type CallFilter = Nothing;
+	type DepositPerItem = DepositPerItem;
+	type DepositPerByte = DepositPerByte;
+	type WeightPrice = pallet_transaction_payment::Pallet<Self>;
+	type WeightInfo = pallet_revive::weights::SubstrateWeight<Self>;
```
Comment (athei) [inline cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs:958]: lets just keep the default weights here like this until we can get rid of the riscv feature

**Comment by pgherveou** [comment]
bot cancel

**Review by athei — APPROVED** [review]

**Comment by athei** [comment]
Just need to run the taplo formatter

**Review by xermicus — APPROVED** [review]

**Pull request merged**